src/extension.ts

```diff
@@ -1431,6 +1439,22 @@ export async function activate(context: vscode.ExtensionContext): Promise<any> {
         DocumentContentProvider.getUri(doc, undefined, undefined, undefined, wsFolder.uri)
       );
     }),
+    vscode.window.tabGroups.onDidChangeTabs((e) => {
+      const processUri = (uri: vscode.Uri): void => {
+        if (uri.scheme == FILESYSTEM_SCHEMA) {
```

Comment:
Are the events not wanted for documents coming from `isfs-readonly` folders?

Reply:
I don't think so. For example, we only show the source control menu for `isfs` files. What do you think @isc-tleavitt?